packages/components/src/components/listitem/listitem.stories.ts

```diff
@@ -183,3 +184,22 @@ export const ListWithIconAndLabels: StoryObj = {
   `),
   ...hideAllControls(),
 };
+
+export const ListWithLongText: StoryObj = {
+  render: () => html`
+    <div role="list" style="width: 15rem">
+      <mdc-listitem label="A short text"></mdc-listitem>
+      <mdc-listitem label="A little lengthy text"></mdc-listitem>
+      <mdc-listitem
+        label="A long lengthy text with small secondary label"
+        secondary-label="a small secondary label"
+      ></mdc-listitem>
+      <mdc-listitem
+        label="A small primary label, A small primary label, A small primary label"
```

Comment:
```suggestion
        label="A small primary label"
```
Shouldn't this be the text if we want a small primary label?

Reply:
If the above change is made, the tooltip is no longer shown on this listitem, which is incorrect.
Can you please take a look?